### ISSUE TITLE
Breaches are not reported correctly, when one source links two documents at the same time

### ISSUE THREAD

**Issue by pgrunewald** [plone/plone.app.linkintegrity#102]
Imagine the following setup:

`/Plone/source`:
```html
<p><a href="/Plone/doc1">Doc1</a></p>
<p><a href="/Plone/doc2">Doc2</a></p>
```

With `/Plone/doc1` and `/Plone/doc2` dwelling peacefully in the Plone site as well.

Now if I try to delete `doc1` and `doc2` and the same time, there should be a warning, that there are **two** breaches, since `/Plone/source` has then two broken links. The current implementation would only report one breach however.

Side note: I have a fix for this in the shelf, but I'd like to wait for #100 to be merged.

**Comment by jensens** [comment]
I merged #100

**Comment by pgrunewald** [comment]
The PR is now ready to be reviewed.
